x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types/crowdstrike.ts

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CROWDSTRIKE_CONNECTOR_ID,
+  SUB_ACTION,
+} from '@kbn/stack-connectors-plugin/common/crowdstrike/constants';
+import { FeaturesPrivileges, Role } from '@kbn/security-plugin/common';
+import SuperTest from 'supertest';
+import expect from '@kbn/expect';
+import { getUrlPrefix } from '../../../../../common/lib';
+import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
+import { createSupertestErrorLogger } from '../../../../../common/lib/log_supertest_errors';
+
+// eslint-disable-next-line import/no-default-export
+export default function createCrowdstrikeTests({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const securityService = getService('security');
+  const log = getService('log');
+  const logErrorDetails = createSupertestErrorLogger(log);
+
+  describe('Crowdstrike', () => {
+    describe('sub-actions authz', () => {
+      interface CreatedUser {
+        username: string;
+        password: string;
+        deleteUser: () => Promise<void>;
+      }
+
+      // Crowdstrike supported sub-actions
+      const crowdstrikeSubActions = [SUB_ACTION.HOST_ACTIONS, SUB_ACTION.GET_AGENT_DETAILS];
+
+      let connectorId: string;
+
+      const createUser = async ({
```

Comment:
hmm. this code looks familiar 😉 😄 

at some point, it may be time to create an FTR service for this stuff and then re-use it across SentinelOne and Crowstrike tests